src/ansys/grantami/bomanalytics/_query_results.py

```diff
@@ -894,13 +915,37 @@ def process(self) -> SustainabilityPhaseSummaryResult:
     @property
     def transport_details(self) -> List[TransportSummaryResult]:
         """
-        Summary information for all transport stages.
+        Summary information for each transport stage described in the BoM.
 
-        Values in percentages express the contribution of the specific transport stage, relative to contributions of all
-        transport stages.
+        Values in percentages express the contribution of the specific transport stage, relative to the total
+        contribution from transportation in the BoM.
         """
         return self._transport_details
 
+    @property
+    def transport_details_grouped_by_category(self) -> List[TransportSummaryByCategoryResult]:
```

Comment:
I don't know if it's better for this to return an empty list or `None` if not provided by the API. If it's `None` then it's more obvious that this isn't a thing the server knows about, but it's irritating that it forces you to do a `None` check before iterating.